README.md

```diff
@@ -13,11 +13,19 @@ def deps(target) do
   [ system(target),
     {:nerves_network, "~> 0.3"}
   ]
+end
+
+# Add nerves_network to your `extra_applications`
+def application(_target) do
+  [
+    extra_applications: [:logger, :nerves_network],
```

Comment:
`:nerves_network` is not needed in the extra applications list. We should probably not list it here